contracts/utils/GsnEip712Library.sol

```diff
@@ -14,7 +14,7 @@ library GsnEip712Library {
     //copied from Forwarder (can't reference string constants even from another library)
     string public constant GENERIC_PARAMS = "address to,bytes data,uint256 value,address from,uint256 nonce,uint256 gas";
 
-    bytes public constant RELAYDATA_TYPE = "RelayData(uint256 gasPrice,uint256 pctRelayFee,uint256 baseRelayFee,address relayWorker,address paymaster)";
+    bytes public constant RELAYDATA_TYPE = "RelayData(uint256 gasPrice,uint256 pctRelayFee,uint256 baseRelayFee,address relayWorker,address paymaster,bytes32 paymasterData,uint256 clientId)";
```

Comment:
add forwarder please please please please

Reply:
Any response here?

Reply:
I still see no reason to expose it - even though we already thrown a lot of garbage into the struct, its no reason to throw out more.